src/avfoundation.cs

```diff
@@ -11481,7 +11481,14 @@ interface AVCaptureDeviceFormat {
 
 		[iOS (8,0), NoMac]
 		[Export ("videoHDRSupported")]
+		bool VideoHDRSupportedVideoHDREnabled { [Bind ("isVideoHDRSupported")] get; }
```

Comment:
This seems odd, can you doublecheck history / Apple docs and see if the name of the property was done this way on purpose? @mandel-macaque may also have some history heh

I think it should be something like
```suggestion
		[Export ("videoHDRSupported")]
		bool IsVideoHdrSupported { [Bind ("isVideoHDRSupported")] get; }
```